src/map.rs

```diff
@@ -441,8 +437,12 @@ impl Map {
 	}
 	pub fn is(&self, r: usize, c: usize, dir: &Direction) -> Option<bool> {
 		match dir {
-			Direction::Up if 0 < r && r < self.rows && c < self.columns => Some(self.is_above(r, c)),
-			Direction::Left if r < self.rows && 0 < c && c < self.columns => Some(self.is_left(r, c)),
+			Direction::Up if 0 < r && r < self.rows && c < self.columns => {
+				Some(self.is_above(r, c))
+			}
+			Direction::Left if r < self.rows && 0 < c && c < self.columns => {
+				Some(self.is_left(r, c))
+			}
```

Comment:
I didn't think these lines were too long. Does Rustfmt have a specific limit somewhere we can change?